zoab_kapoor.cfdg

```diff
@@ -0,0 +1,8 @@
+startshape serpinski
+
+rule serpinski {
+CIRCLE { }
+serpinski { y .255 s .5 b .1 sat .2 }
+serpinski { x -.25 y -.144 s .5 b .1 sat .5 }
+serpinski { x .25 y -.144 s .5 b .1 sat .8 }
+}
```

Comment:
Yeah, I tuned the parameters a bunch until I was happy with it. On higher brightness, the saturation difference is more noticeable, which is kinda cool, but the picture doesn't look as cool.